TWLight/views.py

```diff
@@ -151,36 +153,33 @@ def get_context_data(self, **kwargs):
             event["icon"] = "fa-align-left"
             event["color"] = ""  # grey (default when no color class is applied)
             if app.parent:
+                # fmt: off
                 # Translators: On the website front page (https://wikipedialibrary.wmflabs.org/), this message is on the timeline if a user submits a renewal request. Don't translate <a href=\"{url}\">{partner}</a><blockquote>{rationale}</blockquote>
-                text = _(
-                    "{username} applied for renewal of their "
-                    '<a href="{url}">{partner}</a> access'
-                ).format(
+                text = _("{username} applied for renewal of their <a href='{url}'>{partner}</a> access").format(
                     username=app.editor.wp_username,
                     partner=app.partner.company_name,
                     url=reverse_lazy("partners:detail", kwargs={"pk": app.partner.pk}),
                 )
+                # fmt: on
```

Comment:
This is just a note, not a critique: In other areas, you're unsetting staff interface elements for application management as translatable, but I see we're keeping coordinator application management elements marked. This seems like a very fine line we're cutting that we could easily goof up.

Reply:
That might be true, but this particular instance is setting the context for [/activity/](https://wikipedialibrary.wmflabs.org/activity/), which is public.

Reply:
So it is! I misread the filename!